crates/control/src/motion/look_at.rs

```diff
@@ -29,6 +29,8 @@ pub struct CreationContext {}
 
 #[context]
 pub struct CycleContext {
+    last_motion_command: CyclerState<MotionCommand, "last_motion_command">,
```

Comment:
Can you please elaborate on the circular dependency, you are trying to solve here?
I am not sure whether this is actually the best way to solve this... Nevertheless, this field should definitely not be produced in `look_at`.

Reply:
As it is currently implemented, the production of the `MotionCommand::Initial` with the field `should_look_for_referee` is used as the behavioral trigger to do pose detection. I want to use this field to check whether or not pose detection is enabled in behavior. The motion command is produced in motion which is dependent on behavior, hence the circular dependency. 
Now that I think about it, this decision of whether or not pose detection should run should be done in behavior, which would resolve this issue.